wee_slack.py

```diff
@@ -982,6 +990,7 @@ def __init__(self, eventrouter, token, websocket_url, subdomain, nick, myidentif
         except:
             self.channels = channels
         self.users = users
+        self.external_users = {}
```

Comment:
Is there a reason for creating a separate list for external users instead of placing them in the normal users list?